robots/purePursuitController.h

```diff
@@ -0,0 +1,169 @@
+/*
+    Pure Pursuit Controller. The controller takes a list of waypoints as a path to follow,
+    then it calculates the optimal turning angle to stay on the path
+*/
+
+// BoB robotics includes
+#include "common/main.h"
+#include "robots/car_display.h"
+
+// Third-party includes
+#include "third_party/units.h"
+
+// Standard C++ includes
+#include <chrono>
+#include <iostream>
+#include <thread>
+
+// sign function
+template <typename T> int sgn(T val) {
+    return (T(0) < val) - (val < T(0));
+}
```

Comment:
I'd make this function a private ``static constexpr`` function in your class below just so we don't have stray functions floating around (when you can get name collisions)